dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -552,4 +562,54 @@ public String hostNameUtil(ActionRequest req) {
         return h.getTitle()!=null?h.getTitle():"default";
     }
 
+    /**
+     * Validate if the radio field is valid
+     * @param fieldForm The field form
+     * @param req The action request
+     * @return true if the field is valid, false if not
+     */
+    private boolean isValidRadioField(FieldForm fieldForm, ActionRequest req){
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 5, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1142)